subprojects/workers/build.gradle.kts

```diff
@@ -2,6 +2,8 @@ plugins {
     id("gradlebuild.distribution.api-java")
 }
 
+description = "Infrastructure for running worker processes"
```

Comment:
Do I understand correctly? `:worker-processes` is about everything that happens to start a worker process, while `:workers` is code that runs in those processes. Is this correct?

Reply:
`:worker-processes` is everything that runs in the worker process to load other things the worker may do. This code runs inside the worker. This is the minimum code to get a classloader stood up and communicate with the build process.

`:workers` is the code to start the process. This code runs inside the build process.